client/component-lab/SidebarMachinesListItem/SidebarMachinesListItem.coffee

```diff
@@ -0,0 +1,28 @@
+kd = require 'kd'
+React = require 'react'
+Link  = require 'app/components/common/link'
+
+
+module.exports = class SidebarMachinesListItem extends React.Component
+
+  @propTypes =
+    machine: React.PropTypes.object
+
+  @defaultProps =
+    machine : {}
+
+
+  render: ->
+
+    return  unless @props.machine
```

Comment:
This should not be a concern of this component. If `machine` not present, it shouldn't even be rendered.